health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/Hypothesis.js

```
@@ -1,265 +1,263 @@
+              type={"button"}
+              size={"large"}
+              variation={"primary"}
+              label={t("NO")}
+              onClick={handleCancelDelete}
+            />,
+          ]}
+          sortFooterChildren={true}
+          onClose={() => {
+            setShowPopUp(false);
+          }}
+        ></PopUp>}
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)